src/migrations/20190905134538-create-room.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Rooms', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    facilityId: {
+      type: Sequelize.INTEGER,
+      allowNull: false
+    },
+    roomCount: {
+      type: Sequelize.INTEGER,
+      allowNull: false
+    },
+    roomCategoryId: {
+      type: Sequelize.INTEGER,
+      allowNull: false
+    },
+    createdAt: {
+      allowNull: false,
+      type: Sequelize.DATE
+    },
+    updatedAt: {
+      allowNull: false,
+      type: Sequelize.DATE
+    }
+  }),
+  down: (queryInterface) => queryInterface.dropTable('Rooms')
```

Comment:
I think we would have to include `roomCost` as part of the columns to have. What's your take on the description column for rooms. Here is what I think; Because there is a possibility that a supplier or company facility has a room category that is not included in our list, The description helps us to know how the stakeholder would describe his/her room, giving us a good context for UX

Reply:
I totally agree with you on this, would make a push and update that.

Reply:
Done

Reply:
Good job @King-Success. I taught it may be better if the cost was of datatype Decimal(10,2). I recently updated my implementation with that.

Reply:
Since we would be sharing the same tables. You may also update the `allowNull: true` for roomCost

Reply:
Ok, I've updated as suggested